system/Database/OCI8/Builder.php

```diff
@@ -48,16 +48,6 @@ class Builder extends BaseBuilder
      */
     protected $countString = 'SELECT COUNT(1) ';
 
-    /**
-     * Limit used flag
-     *
-     * If we use LIMIT, we'll add a field that will
-     * throw off num_fields later.
-     *
-     * @var bool
-     */
-    protected $limitUsed = false;
```

Comment:
since it not exists on parent, this need to be kept, with mark `@deprecated` for patch release.

Reply:
I thought about it, but I realized it never had any use. It was simply a copy-paste from CI3 without the full implementation (in the `Result::getFieldCount()`), which makes it non-usable. It was a "dead" code from the beginning.

But I'm happy to make changes if more people share your view.